src/main/java/Duke.java

```diff
@@ -1,10 +1,261 @@
+                                    Deadline newDeadline = new Deadline(deadlineDesc, deadline);
+                                    System.out.println(" " + newDeadline.toString());
+                                    myList.add(newDeadline);
+                                    System.out.println("Now you have " + myList.size() + " tasks in the list.");
+                                    database.writeTaskToFile(myList);
+
+                                }
+                            } catch(DukeException e) {
+                                System.out.println(e.getMessage());
+                            }
+                        }
+
+                    } catch (DukeException e){
+                        System.out.println(e.getMessage());
+                    }
+                    System.out.println("____________________________________");
+                    break;
+
+                case "event":
+
+                    System.out.println("____________________________________");
+                    try{
+                        String s1 = input.nextLine();
+                        if(s1.equals("")){
+                            throw new DukeException("Enter valid Event.");
+                        } else {
+                            try {
+                                String wholeString = s1.substring(1);
+                                String[] parts = wholeString.split(" /at ");
+                                String eventDesc = parts[0];
+                                if(parts.length == 1){
+                                    throw new DukeException("Please adhere to convention:\n(Event /at event details)");
+                                } else {
+                                    String eventDetails = parts[1];
+                                    System.out.println("Got it. I've added this Event:");
+                                    Event newEvent = new Event(eventDesc, eventDetails);
+                                    System.out.println(" " + newEvent.toString());
+                                    myList.add(newEvent);
+                                    System.out.println("Now you have " + myList.size() + " tasks in the list.");
+                                    database.writeTaskToFile(myList);
+                                }
+                            } catch(DukeException e) {
+                                System.out.println(e.getMessage());
+                            }
+                        }
+
+                    } catch (DukeException e){
+                        System.out.println(e.getMessage());
+                    }
+                    System.out.println("____________________________________");
+                    break;
+
+                case "list":
+                    System.out.println("____________________________________");
+                    System.out.println("Here are the tasks in your list:");
+                    for(int i=1; i< myList.size()+1; i++){
```

Comment:
Note the spaces here as well:
```suggestion
                    for (int i = 1; i < myList.size() + 1; i++) {
```